### ISSUE TITLE
Fine-tune CBraMod on sleep-edf-20 for single channel sleep staging task can't converge

### ISSUE THREAD

**Issue by ejejeieei** [wjq-learning/CBraMod#20]
Thank you for your fantastic work. I meet a problem when I fine-tune CBraMod on sleep-edf-20 using 1 to 1 scheme: it doesn't converage at all. Could you give me some suggestion about this? Thanks!
Optim: adamw
epoch: 40
learning rate: 1e-4
lr_scheduler: onecycle
gpu: rtx 2080
5-fold cross validation
20 subjects in sleep-edf-20
